changelog.md

```diff
@@ -7,10 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Releases]
 
+## [1.5.1] - 2022-12-19
+### Added
```

Comment:
Section should be **Changed** in that case

Reply:
Related to that, I will request if possible to properly update the 1.5.0 entry with two entries:
- Added -> .net 7 support
- Changed -> current sdk's dependencies upgrade.

Many thanks @javierpardollama !!

Reply:
Done @Xelit3